articles/beginner-guide-to-npm/index.md

```diff
@@ -0,0 +1,577 @@
+Every NPM package will be installed in the `node_module` folder.  `npm install <module>` installs the latest package version available in the NPM registry. Open the `package.json` file. Apart from the project aspect, we initialize with `init`, the following lines were added.
+
+```bash
+  "dependencies": {
+    "express": "^4.17.1",
+    "lodash": "^4.17.20"
+  }
+  ```
+
+The packages we installed are saved as an entry in the dependencies field. The dependencies field save any package required by your app to run. It is possible to save a package as devDependencies. A flag `--save-dev` is added to the `npm install  <module>`. This will save the package as devDependencies. DevDependencies are the packages that are not required by the app to run. These packages are used for development purposes such as testing, local server speeding for development purposes, transpiring code, etc.
+
+Usage:
+
+```bash
+npm install nodemon --save-dev ## install a dev package to our dependencies
+```
+
+```bash
+$ npm install nodemon --save-dev
+
+> nodemon@2.0.4 postinstall E:\myapp\node_modules\nodemon
+> node bin/postinstall || exit 0
+
++ nodemon@2.0.4
+added 119 packages from 53 contributors and audited 121 packages in 41.629s
+
+10 packages are looking for funding
+  run `npm fund` for details
+
+found 0 vulnerabilities
+```
+
+A devDependencies scaffold will be added to the package.json file.
+
+```bash
+ },
+  "devDependencies": {
+    "nodemon": "^2.0.4"
+  }
+```
+
+#### Package.json Dependencies Management
+The main aim of using `package.json` is to define your application's dependencies. For instance, if you have cloned someone else project to your local computer, this project will not execute as intended. You have not installed the necessary packages to run this app. This project has a `package.json` file with a dependencies scaffold that indicates the packages the project needs. Thus, you need to go ahead and install these packages. Assuming the project has several packages, it might be tiresome to do that. NPM resolves this hassle for you. Run `npm i` (alias `i` replace install) inside the cloned project's root directory. `npm i` fetch plus install individual needed packages specified in the project `package.json` dependencies.
+
+For example:
+
+To understand this, install several packages in the project we created at the beginning of this guide.
+Go ahead and install the following packages.
+
+```bash
+npm i moment
+npm i underscore
+npm i mongoose
+```
+
+The command below installs multiple packages.
+
+```bash
+npm i mongoose  moment underscore ## install multiple packages at a go
+```
+
+Now, assume you have cloned this project from another computer, and you want to install the colonies of this project. Go ahead and delete the `node_module` folder of your project. This will remove the packages installed on your local system. If you run `npm list --depth=0` to list down the packages installed for this project, you will get something close to this:
```

Comment:
Just making sure this was not a typo:
"install the colonies" 

Not sure

Reply:
Wasn't a typo.
Made it simpler